### ISSUE TITLE
Improve the scope of tactical hud functionality.

### ISSUE THREAD

**Issue by X0-11** [HaloSpaceStation/HaloSpaceStation13#850]
Ideally tactical huds would do more than allow you to see waypoints. They'd be able to show a readout of squadmembers, their pulse, perhaps even ammo readouts in weapons.

**Comment by Max-023** [comment]
Lit. Always wondered why we had them when they did nothing.

I do think it should be given to combat roles tho, and given a little sprite screen so even if a Marine has no visor he would still have something on him. The noncombat roles aren't in situations that would practically use this.

**Comment by BDpuffy420** [comment]
They currently do work just for seeing waypoints, the thing is who ever has the squad waypoint tool has to add you to the squad first

**Comment by Maxattacker** [comment]
Maybe instead of more gui elements, put it in a tab.

**Comment by Max-023** [comment]
Yeah I know squad waypoints are for that, but they are rarely used.

**Comment by Maxattacker** [comment]
It would be better to give a readout of who is in your squad, and possibly putting in the bpm and ammo counter, but to make those stay updated constantly sounds like it would be a lot of work.

**Comment by BDpuffy420** [comment]
this has been done with squad way point manager plus being able to call down supply drops with them now.
